10.11/Dockerfile.rhel10

```diff
@@ -0,0 +1,73 @@
+FROM registry.stage.redhat.io/ubi10/s2i-core
+
+# MariaDB image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MariaDB
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
+
+ENV MYSQL_VERSION=10.11 \
+    APP_DATA=/opt/app-root/src \
+    HOME=/var/lib/mysql \
+    NAME=mariadb \
+    VERSION=10.11 \
+    SUMMARY="MariaDB 10.11 SQL database server" \
+    DESCRIPTION="MariaDB is a multi-user, multi-threaded SQL database server. The container \
+image provides a containerized packaging of the MariaDB mysqld daemon and client application. \
+The mysqld server daemon accepts connections from clients and provides access to content from \
+MariaDB databases on behalf of the clients."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="MariaDB $VERSION" \
+      io.openshift.expose-services="3306:mysql" \
+      io.openshift.tags="database,mysql,mariadb,mariadb1011,mariadb-1011" \
+      com.redhat.component="$NAME-1011-container" \
+      name="rhel10/mariadb-1011" \
+      version="1" \
+      usage="podman run -d -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306 rhel10/mariadb-1011" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+EXPOSE 3306
+
+# This image must forever use UID 27 for mysql user so our volumes are
+# safe in the future. This should *never* change, the last test is there
+# to make sure of that.
+RUN INSTALL_PKGS="policycoreutils rsync tar xz gettext hostname bind-utils groff-base mariadb-server" && \
```

Comment:
Why did we move from installing xz in s2i-core to installing it here?

Reply:
I guess, it was done for CentOS Stream 10, to have image smaller.
The corresponding pull request is here: https://github.com/sclorg/s2i-base-container/pull/306 and especially comment: 
https://github.com/sclorg/s2i-base-container/pull/306#issuecomment-2236591698

Reply:
Makes sense in that case, thanks for pointing me in the right direction!